packetery/template/checkout/checkout_script.latte

```diff
@@ -0,0 +1,44 @@
+<script>
+	jQuery(function ($) {
+		$.getScript("https://widget.packeta.com/v6/www/js/library.js")
+
+		var $widgetDiv = $('.packeta-widget');
+
+		$(document).on('change', '#shipping_method input[type="radio"]', function () {
+			if (this.value.startsWith('packetery_shipping_method')) {
+				$widgetDiv.show();
+			} else {
+				$widgetDiv.hide();
+				// clear selected pickup point
+				$widgetDiv.find('.packeta-pickup-point-name').empty();
+
+				// clear hidden inputs
+				{foreach $pickup_point_attrs as $attr}
+				$('#{$attr['name']|noescape}').val('');
+				{/foreach}
+			}
+		});
+
+		$('.packeta-widget-button').click(function (e) {
+			e.preventDefault();
+			var widgetOptions = {
+				appIdentity: {$app_identity},
+				country: $widgetDiv.data('country'),
```

Comment:
Nechceme už rovnou předpokládat více možností? Ušetří to práci v nadcházejících ticketech. A nebo tam packeta-widget může být jen jednou?

`$(e.target).closest('.packeta-widget').data('country')`

Reply:
Zatím tam může být jednou. Pokud by to někdy mohlo být jinak, změníme selector.